forloop_modules/function_handlers/auxilliary/form_dict_list.py

```diff
@@ -128,10 +138,10 @@ def button(self, function, function_args, text, focused: bool = False, enforce_r
             row (int): row number of the element - specified only if it's not first element on the same row
         """    
         key = "Button"
-        value = {"name": name, "function": function, "function_args": function_args, "text": text, "focused": focused, "enforce_required": enforce_required, "frontend_implementation": frontend_implementation}
+        value = {"name": name, "function": function, "function_args": function_args, "text": text, "focused": focused, "enforce_required": enforce_required, "frontend_implementation": frontend_implementation, 'desktop_only': desktop_only}
```

Comment:
frontend_implementation + desktop_only are duplicates

Reply:
I reached out to Tomas and Joaquin for clarification - I believe `frontend_implementation` stores the information if the button should be implemented in webapp with custom logic. `desktop_only` stores the information if the buttons should be rendered in webapp.